pom.xml

```diff
@@ -1,12 +1,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    
+
     <groupId>io.cloudsoft.aws.elb</groupId>
-    <artifactId>brooklyn-aws-elb</artifactId>
+    <artifactId>brooklyn-aws-elb-parent</artifactId>
     <version>0.6.0-SNAPSHOT</version> <!-- BROOKLYN_AWS_ELB_VERSION -->
```

Comment:
Can you add a commit before all of the others that changes the version to 0.7.0-SNAPSHOT? Once this is merged I'll make a 0.6.0 branch without these changes that has a dependency on the 0.11.0 Brooklyn release.